### ISSUE TITLE
skip-row isn't skipping all rows that satisfy the conditions

### ISSUE THREAD

**Issue by bhatiadivij** [usc-isi-i2/t2wml#88]
Please check this example. The skip-row isn't skipping row 9.
But If I try to skip rows with `value[D, $row] == 2` it works as expected. I think the issue might be with trimming the cell values.
![image](https://user-images.githubusercontent.com/25511720/81312172-2ebf9880-903b-11ea-887e-bb1542c5b6b1.png)
Here is the sample YAML file based on Homicide data Table-1a:
```yaml
statementMapping:
  region:
    - left: D
      right: F
      top: 4
      bottom: 9
      skip_row:
        - value[D, $row] == 1
  template:
    item: item[A, $row]
    property: P100024 # murder
    value: value[$col, $row]
    #unit: D1002
    qualifier:
      - property: P585
        value: value[$col, 3]
        calendar: Q1985727
        precision: year
        time_zone: 0
        format: "%Y"
      - property: P6001 # applies to people
        value: item[C, $row]
      - property: P123 #source
        value: item[B, $row]

```

**Comment by bhatiadivij** [comment]
Similar behavior is observed with `skip_column` as well.
![image](https://user-images.githubusercontent.com/25511720/81335601-b8339280-905c-11ea-8571-beab146ee186.png)

**Comment by devowit** [comment]
fixed in commit 892e289
